core/src/main/java/com/google/zxing/pdf417/encoder/PDF417HighLevelEncoder.java

```diff
@@ -174,16 +174,15 @@ static String encodeHighLevel(String msg, Compaction compaction, Charset encodin
     if (msg.isEmpty()) {
       throw new WriterException("Empty message not allowed");
     }
+    
+    if (Compaction.TEXT == compaction) {
+      checkCharset(msg,127,"Consider specifying Compaction.AUTO instead of Compaction.TEXT");
```

Comment:
The logic looks OK; minor request though: everywhere in the change, follow a comma with a space for consistency